infra/src/main/kotlin/fi/fta/geoviite/infra/ratko/model/RatkoAssetModel.kt

```diff
@@ -95,7 +95,6 @@ enum class RatkoAssetState(@get:JsonValue val value: String, val category: Layou
     @Suppress("unused")
     SUGGESTED("SUGGESTED", LayoutStateCategory.FUTURE_EXISTING),
 
-    @Suppress("unused")
```

Comment:
Unused no more! 🚀 Harvoin on tullut eteen tilanteita joissa joku aiemmin suunniteltu ja koodattu mutta käyttämätön juttu on oikeasti otettu käyttöön, pitkästä aikaa näin päin taas :D